src/interfaces/service.interface.ts

```diff
@@ -0,0 +1,4 @@
+export interface ServiceInterface {
+    getGames(): Promise<string>
+    getPrices(): Promise<string>;
```

Comment:
We don't need prices. I don't get this information from Epic Games

Reply:
Fixed by 7a8ef43b52df382ca8af5451b7e23e31421f126b